README.md

```diff
@@ -105,7 +105,22 @@ To ensure the SDK is functioning correctly, we have a comprehensive test suite.
 
 ## Release Process
 
-This repository uses a two-step workflow process for creating new releases:
+### Releasing a new version for development
+
+Once a package is ready for a new version, we first publish a dev version and test it in local environment.
+
+1. Run `npm version prerelease --preid=dev` under either `packages/sdk-js` or `packages/types` to update the version in `package.json`.
+2. Run `npm publish --tag dev` under either `packages/sdk-js` or `packages/types` to publish the new dev version to NPM. Most importantly, this **bumps up version number** in `package.json` of `packages/types`.
+3. If the `types` package has a new version, since it is depended on by `sdk-js`, we need to make sure `sdk-js` can build with the new version.
+
+   1. `yarn run clean` **at the root folder** to remove existing node_modules folder and yarn.lock file.
+   2. Run `yarn install` under the root folder to re-install the dependencies. You should see a prompt asking the version of `@avaprotocol/types` to install. Choose the new version you just created in step 1.
+   3. Run `yarn build` under the root folder to build all packages.
```

Comment:
@v9n could you try these commands again to see if packages could build? The key is to remove the last version of `types` package from `yarn.lock`(in the clean command)
`yarn run clean`
`yarn install`
`yarn build`